01/app.js

```diff
@@ -1 +1,21 @@
-console.log('DOM');
\ No newline at end of file
+console.log('DOM');
+
+
+
+const item = document.getElementsByClassName('.comments__item comments__item--newest');
```

Comment:
niestety to rozwiązanie nie działa i pokazuje 0 elementów, poza tym chcesz wyszukać 1 element a nie wiele
powinno być `const item = document.querySelector('.comments__item.comments__item--newest')`

Reply:
mi pokazuje 5, dlaczego ?